src/python_testing/TC_MCORE_FS_1_2.py

```diff
@@ -0,0 +1,185 @@
+#
+#    Copyright (c) 2024 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# See https://github.com/project-chip/connectedhomeip/blob/master/docs/testing/python.md#defining-the-ci-test-arguments
+# for details about the block below.
+#
+
+import ipaddress
+import logging
+import os
+import pathlib
+import queue
+import random
+import signal
+import subprocess
+import time
+import uuid
+
+import chip.clusters as Clusters
+from chip import ChipDeviceCtrl
+from chip.interaction_model import InteractionModelError, Status
+from matter_testing_support import MatterBaseTest, TestStep, async_test_body, default_matter_test_main, has_cluster, per_endpoint_test
+from mobly import asserts
+from TC_SC_3_6 import AttributeChangeAccumulator
```

Comment:
please use the one from support. AttributeChangeCallback - it manages the queue for you and you can just call wait_for_report

Reply:
I don't think that supports the use case here where we are subscribed to a wildcard set of endpoints.

Reply:
Actually, I think I was confusing the callback from the sub. I think this works. And, I can defer to the timeout here, I don't' think I need the long timeouts from RR

Reply:
Actually, it's missing a few other things. CLosing for now and will follow-up with improvements to the common utilities.